pkg/test/common.go

```diff
@@ -184,6 +186,14 @@ func MakeHtpasswdFile() string {
 	return MakeHtpasswdFileFromString(content)
 }
 
+func GetLatestTag(allTags []cvemodel.TagInfo) cvemodel.TagInfo {
```

Comment:
Why not move this to `pkg/test/oci-layout/oci_layout.go`? Is it used somewhere else?

Reply:
It is also used in pkg/extesions/search